app/views/images/crop.html.erb

```diff
@@ -34,7 +34,7 @@ content_for :title,t(title_key, title: @edition.title_or_fallback)
 
   <div class="app-js-only">
     <%= render "govuk_publishing_components/components/button", {
-      text: (@image_revision.at_exact_dimensions?) ? "Continue" : "Crop image",
+      text: (@image_revision.at_exact_dimensions?) || params[:wizard] != 'upload' ? "Continue" : "Crop image",
```

Comment:
Should this conditional be the other way around?

Reply:
Doh, yes you're correct!